packages/ariakit/src/collection/__examples__/collection/style.css

```diff
@@ -0,0 +1,22 @@
+.collection {
+  @apply
+    overflow-hidden
+    border-none
+    rounded-lg;
```

Comment:
```suggestion
  @apply flex flex-col gap-2;
```

Reply:
Done in https://github.com/reakit/reakit/pull/977/commits/4f909150cb671cf1738a9a279e2b5be691898b7f